ONSdigital/dp-nlp-category-api
# Feature/enhance local running

### What

- Removed unused make targets
- Fixed application level logging
- Fixed unit tests so they can run on macos
- Improved readme
- Improved clarity on env vars and env file generation

### How to review

Review code, check makes sense and tests pass. 
Should be able to use readme to run application locally.
To test functional changes you can:
- check for logger.info messages appearing now (e.g. startup config) 
- trigger an error by adding an exception to one of the `routes` and see if it logs valid json (it wasn't before)

### Who can review

Not me.
